### PR TITLE
Write createReviews script to seed databases

### PR DESCRIPTION
**Pull request by willmainwaring**

### CODE REVIEW

**Review by keenanthomson — COMMENTED** [review]
This looks good to me - I think it's mergeable, but two questions:

- could you reduce your createReviews to a single script? (ie. both cassandra and mysql pull from the same .csv)
- do you need the references to 'data.json' in the seed script? I couldn't confirm it exists, and it looks like the script just runs the INFILE command for your .csv

File: database/cassandra/createReviews.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+const images = JSON.parse(fs.readFileSync('database/mysql/images.json') + '');
+const createReview = (cb, id) => {
+  const review = {};
+  review.name = faker.name.firstName();
+  review.id = id;
+  review.avatar = `https://sdc-reviews-avatars.s3.us-east-2.amazonaws.com/${images[faker.random.number({ min: 0, max: 404 })]}`;
+  review.date = faker.date.past(10);
+  review.entry_id = faker.random.number({ min: 1, max: 1000000 });
```
Comment (ruparocks) [inline database/cassandra/createReviews.js]: Is the entry_id a reference to the property id?

File: database/mysql/createReviews.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	review.name = faker.name.firstName();
+	review.avatar = `https://sdc-reviews-avatars.s3.us-east-2.amazonaws.com/${images[faker.random.number({ min: 0, max: 404 })]}`;
+	review.date = faker.date.past(10);
+	review.entry_id = faker.random.number({ min: start, max: end });
+	review.content = faker.lorem.sentences(faker.random.number({ min: 1, max: 10 }));
+	cb(review);
+};
+
+const saveReviews = (start, end) => {
+	for (var i = start; i < end; i++) {
```
Comment (ruparocks) [inline database/mysql/createReviews.js]: You could try to write more than one record to the csv at a time, just a recommendation.

File: database/mysql/schema.sql [hunk 3]
```diff
@@ -6,9 +6,10 @@ USE airbnb;
 
 CREATE TABLE reviews (
   id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  entry_id INT NOT NULL,
   name VARCHAR(25) NOT NULL,
-  avatar VARCHAR(25) NOT NULL,
-  numDaysAgo INT NOT NULL,
+  avatar VARCHAR(75) NOT NULL,
+  date  VARCHAR(30) NOT NULL,
```
Comment (ruparocks) [inline database/mysql/schema.sql]: Date is a varchar? Why is this not a Datetime?

**Review by ruparocks — COMMENTED** [review]
Clean code. I left a few questions if you can comment back and I understand your choices, I will approve the PR.

File: database/cassandra/createReviews.js (hunk 1 above)
Comment (willmainwaring) [reply database/cassandra/createReviews.js]: Yes, it is. I realized I should set the max value based on the number of reviews I'm creating, so I replaced it with a variable.

File: database/mysql/schema.sql (hunk 3 above)
Comment (willmainwaring) [reply database/mysql/schema.sql]: I was only interested in the year/month/day in the date, and I was having some trouble figuring out which date format would work with mySQL, so I converted it to a string with a plan to slice off the first x characters.

I think you're right, though, so I'm changing this to datetime.

**Comment by willmainwaring** [comment]
@keenanthomson 

Goods suggestions! I've simplified to one `createReviews` that accepts an id as an argument and will conditionally create a matching key for each review.

Also, `seed.js` was an old file, so I deleted it.

**Pull request merged**